src/openloco/windows/build_vehicle.cpp

```diff
@@ -1,27 +1,283 @@
 #include "../companymgr.h"
+#include "../game_commands.h"
+#include "../graphics/colours.h"
+#include "../graphics/image_ids.h"
 #include "../input.h"
 #include "../interop/interop.hpp"
+#include "../localisation/FormatArguments.hpp"
+#include "../objects/cargo_object.h"
+#include "../objects/interface_skin_object.h"
+#include "../objects/objectmgr.h"
+#include "../objects/road_extra_object.h"
+#include "../objects/road_object.h"
+#include "../objects/track_extra_object.h"
+#include "../objects/track_object.h"
+#include "../objects/vehicle_object.h"
 #include "../openloco.h"
 #include "../things/thingmgr.h"
 #include "../ui/WindowManager.h"
+#include "../ui/scrollview.h"
+#include "../widget.h"
 
 using namespace openloco::interop;
 
 namespace openloco::ui::build_vehicle
 {
-    static loco_global<int16_t, 0x01136268> _1136268;
-    static loco_global<uint16_t[1], 0x0113626A> _113626A;
-    static loco_global<int32_t, 0x011364E8> _11364E8;
-    static loco_global<int8_t[1], 0x011364F0> _11364F0;
+    static const gfx::ui_size_t window_size = { 380, 233 };
+
+    enum widx
+    {
+        frame = 0,
+        caption = 1,
+        close_button = 2,
+        panel = 3,
+        tab_build_new_trains,
+        tab_build_new_buses,
+        tab_build_new_trucks,
+        tab_build_new_trams,
+        tab_build_new_aircraft,
+        tab_build_new_ships,
+        tab_vehicles_for_0,
```

Comment:
`track_tab`. Go through all of this and conform to track_tab and track type for this set of tabs.